plugins/modules/rabbitmq_exchange.py

```diff
@@ -13,52 +13,54 @@
 module: rabbitmq_exchange
 author: Manuel Sousa (@manuel-sousa)
 
-short_description: Manage rabbitMQ exchanges
+short_description: Manage rabbitMQ exchange
 description:
   - This module uses rabbitMQ Rest API to create/delete exchanges
 requirements: [ "requests >= 1.0.0" ]
 options:
     name:
         description:
-            - Name of the exchange to create
+            - Name of the exchange to create.
         type: str
         required: true
     state:
         description:
-            - Whether the exchange should be present or absent
+            - Whether the exchange should be present or absent.
         type: str
         choices: [ "present", "absent" ]
         required: false
         default: present
     durable:
         description:
-            - whether exchange is durable or not
+            - Whether exchange is durable or not.
         type: bool
         required: false
         default: true
     exchange_type:
         description:
-            - type for the exchange
+            - Type for the exchange.
+            - If using I(x-random), I(x-consistent-hash) or I(x-recent-history) the respective plugin on the RabbitMQ server
```

Comment:
Do we need to add `x-delayed-message` here?

Reply:
Great point, good catch. `x-delayed-message` was already included in the codebase and I cannot see it in https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/ , although I found it here: https://github.com/rabbitmq/rabbitmq-delayed-message-exchange